docs/source/bmi.grid_funcs.rst

```diff
@@ -21,6 +24,9 @@ Grid Type
 
   string get_grid_type(in int id)
 
+..
+   Use "String" instead of "string" so that your syntax highlighter catches it?
+
```

Comment:
The `string` type is Scientific Interface Definition Language (SIDL), for which we don't have a pygments lexer. We're using the java lexer, which is close. I haven't found a better one.